ocaml/idl/datamodel_host.ml

```diff
@@ -373,6 +374,13 @@ let evacuate =
         ; param_release= next_release
         ; param_default= Some (VRef null_ref)
         }
+      ; {
+          param_type= Int
+        ; param_name= "evacuate_batch_size"
+        ; param_doc= "The maximum number of VMs to be migrated per batch"
```

Comment:
Should this include something like "Zero (0) will use a default from xapi.conf"? Otherwise the default is surprising.

Reply:
Done :)